src/System.Net.HttpListener/tests/Resources/System.Net.HttpListener.Tests.rd.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Directives xmlns="http://schemas.microsoft.com/netfx/2013/01/metadata">
+  <Library Name="System.Net.Tests.HttpListenerRequestTests">
+    <Namespace Name="System.Text" Dynamic="Required All" />
```

Comment:
> is there a way to enable it only for UTF8Encoding?
For that field, we can only put Namespace or Assembly. I don't know how to enable only for one class.

@tijoytom could you please answer Cristian's question?